### PR TITLE
Develop/feature/author page

### PR DESCRIPTION
**Pull request by sahilsaini787**
Added author page (styling remaining)

### CODE REVIEW

File: src/Components/ArticlePreviewSection/ArticlePreviewSection.tsx [hunk 1]
```diff
@@ -1,57 +1,25 @@
...
+        {posts ? (
+          posts.map((articleData: PostType) => (
+            <ArticleCard
+              key={articleData.id}
+              postData={articleData}
+              layoutStyle={layoutStyle}
+            />
           ))
         ) : (
           <span>Loading Articles...</span>
```
Comment (melon-husk) [inline src/Components/ArticlePreviewSection/ArticlePreviewSection.tsx:25]: Still required?

File: src/app/[id]/page.tsx [hunk 2]
```diff
@@ -1,30 +1,23 @@
...
-
-const DisplayPostsByCategory = async ({ params }: ParamsType) => {
-  const id: string = (await params).id;
-  if (!id) {
-    notFound();
-  }
+const DisplayPosts = async ({ params }: ParamsType) => {
   const categories: CategoriesType = await fetchCategories();
-
+  const id = (await params).id;
```
Comment (melon-husk) [inline src/app/[id]/page.tsx:12]: I think you meant to use `[slug]` here instead of id

**Review by melon-husk — CHANGES_REQUESTED** [review]

File: src/Components/ArticlePreviewSection/ArticlePreviewSection.tsx (hunk 1 above)
Comment (sahilsaini787) [reply src/Components/ArticlePreviewSection/ArticlePreviewSection.tsx:25]: Only for testing during development. Will be removed in final build.

File: src/app/[id]/page.tsx (hunk 2 above)
Comment (sahilsaini787) [reply src/app/[id]/page.tsx:12]: I renamed slug to id.

**Pull request merged**